badger_db.go

```diff
@@ -28,7 +28,12 @@ func NewBadgerDB(dbName, dir string) (*BadgerDB, error) {
 	if err := os.MkdirAll(path, 0755); err != nil {
 		return nil, err
 	}
+
+
 	opts := badger.DefaultOptions(path)
+	opts.IndexCacheSize = 256 << 20 // 100 mb or some other size based on the amount of data
```

Comment:
I'm fine with the idea of setting some reasonable baseline value here, but we should document where it came from.

According to Badger's docs, the Bloom filter for each SSTable is 5MB (presumably MiB). Do we have any data about typical table counts for production networks?  (We don't need to tune this super precisely, but the reader ought to have _some_ idea of where we got the value)

Reply:
I don't, but I like where this is heading and I'll get that in here, too.